### PR TITLE
docs: Fixed starship config location

### PR DESCRIPTION
**Pull request by BRAVO68WEB**
#### Description
config file during creation and when used as export variable into shells are different.
Fixed to make it uniform and avoid errors

#### Motivation and Context
I was installing starship today while reading the documentation for the same.
The configuration file location while creation and when it was exported to shells were different.
[here](https://starship.rs/config/#configuration)  

#### Checklist:
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

**Comment by davidkna** [comment]
To fix the Semantic Pull Request check, you can either add another commit with any commit message or change the first commit message to conform to semantic commit guidelines by prepending `docs: ` to the commit message, e.g.

```
git commit --allow-empty -m 'make Semantic PR bot happy'
git push 
```

**Review by andytom — CHANGES_REQUESTED** [review]
This is incorrect the default starship configuration file is `~/.config/starship.toml`, I think the confusion is coming from the example of using the `STARSHIP_CONFIG` environment variable to set a different location for the configuration file.

**Comment by matchai** [comment]
This isn't the first time that example has lead to confusion. I think it's time we change the example to be distinctly intentionally different.

**Comment by BRAVO68WEB** [comment]
Ok I am do the requested changes

File: docs/config/README.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ disabled = true
 You can change default configuration file location with `STARSHIP_CONFIG` environment variable:
 
 ```sh
-export STARSHIP_CONFIG=~/.starship/config.toml
+export STARSHIP_CONFIG=~/.config/starship.toml
```
Comment (andytom) [inline docs/config/README.md:27]: I think this should be something like

```suggestion
export STARSHIP_CONFIG=~/example/non/default/path/starship.toml
```

To make it explicit that this is just an example.

File: docs/config/README.md [hunk 2]
```diff
@@ -24,19 +24,19 @@ disabled = true
...
 ```
 
 Equivalently in PowerShell (Windows) would be adding this line to your `$PROFILE`:
 
 ```powershell
-$ENV:STARSHIP_CONFIG = "$HOME\.starship\config.toml"
+$ENV:STARSHIP_CONFIG = "$HOME\.config/starship.toml"
```
Comment (andytom) [inline docs/config/README.md:33]: And here

```suggestion
$ENV:STARSHIP_CONFIG = "$HOME\example\non\default\path\starship.toml"
```

File: docs/config/README.md [hunk 3]
```diff
@@ -24,19 +24,19 @@ disabled = true
...
 ```
 
 Or for Cmd (Windows) would be adding this line to your `starship.lua`:
 
 ```lua
-os.setenv('STARSHIP_CONFIG', 'C:\\Users\\user\\.starship\\config.toml')
+os.setenv('STARSHIP_CONFIG', 'C:\\Users\\user\\.config\\starship.toml')
```
Comment (andytom) [inline docs/config/README.md:39]: Here too

```suggestion
os.setenv('STARSHIP_CONFIG', 'C:\\Users\\user\\example\\non\\default\\path\\starship.toml')
```

**Comment by BRAVO68WEB** [comment]
Added suggested changes

**Review by andytom — APPROVED** [review]

**Pull request merged**